### DISCUSSION TITLE
Miryoku ZMK Supported Keyboards and ZMK Out-of-Tree Keyboards Directory

### DISCUSSION THREAD

**Discussion by manna-harbour** [manna-harbour/miryoku#81] [Announcements]
**This is the defacto ZMK out-of-tree keyboards directory. For use outside of Miryoku ZMK, checkmarks and strikethrough can be ignored.**

Checked keyboards are supported in Miryoku ZMK. Support for unchecked keyboards will be added upon [request](https://github.com/manna-harbour/miryoku/discussions/new?category=ideas). Struck out keyboards are not applicable (numpads and macropads), or have been superseded by another entry. Also see the [Miryoku ZMK Supported Keyboards](https://github.com/manna-harbour/miryoku_zmk#supported-keyboards) documentation. 

###  Onboard Controller Keyboards

- [ ] ~~BDN9 Rev2 (Board: bdn9_rev2)~~
- [ ] BT60 V1 Hotswap (Board: bt60_v1_hs) [60%]
- [ ] BT60 V1 Soldered (Board: bt60_v1) [60%]
- [x] Corneish Zen v2 (Boards: corneish_zen_v2_left, corneish_zen_v2_right) [ergo split]
- [x] Ferris 0.2 (Board: ferris_rev02)
- [x] Glove80 (Boards: glove80_lh, glove80_rh) [ergo split, note non-standard names]
- [x] nice!60 (Board: nice60)
- [x] Planck Rev6 (Board: planck_rev6)
- [ ] Preonic Rev3 (Board: preonic_rev3) [ortho]
- [ ] S40NC (Board: s40nc) [40%]

#### Pro Micro Interconnect

- [x] A. Dux (Shields: a_dux_left, a_dux_right)
- [x] BAT43 (Shield: bat43)
- [ ] BFO-9000 (Shields: bfo9000_left, bfo9000_right) [ortho split]
- [ ] ~~Boardsource 3x4 Macropad (Shield: boardsource3x4)~~
- [x] Boardsource 5x12 (Shield: boardsource5x12)
- [ ] Chalice (Shield: chalice) [alice]
- [x] Clog (Shields: clog_left, clog_right) [also for use with sephirette, clog v3]
- [x] Contra (Shield: contra)
- [x] Corne (Shields: corne_left, corne_right) [also for use with besom, fifi, chocofi, etc.]
- [x] Cradio/Sweep (Shields: cradio_left, cradio_right)
- [x] CRBN Featherlight (Shield: crbn)
- [x] eek! (Shield: eek)
- [x] Elephant42 (Shields: elephant42_left, elephant42_right)
- [x] Ergodash (Shields: ergodash_left, ergodash_right)
- [ ] ~~Eternal Keypad Lefty (Shield: eternal_keypad_lefty)~~
- [ ] ~~Eternal Keypad (Shield: eternal_keypad)~~
- [ ] Fourier Rev. 1 (Shields: fourier_left, fourier_right) [split 40%]
- [x] Helix (Shields: helix_left, helix_right)
- [x] Iris (Shields: iris_left, iris_right)
- [x] Jian (Shields: jian_left, jian_right)
- [x] Jiran (Shields: jiran_left, jiran_right)
- [x] Jorne (Shields: jorne_left, jorne_right)
- [ ] ~~Knob Goblin (Shield: knob_goblin)~~
- [x] Kyria Rev2 (Shields: kyria_rev2_left, kyria_rev2_right)
- [x] Kyria (Shields: kyria_left, kyria_right)
- [x] Leeloo (Shields: leeloo_left, leeloo_right)
- [x] Lily58 (Shields: lily58_left, lily58_right)
- [x] Lotus58 (Shields: lotus58_left, lotus58_right)
- [x] Microdox (Shields: microdox_left, microdox_right)
- [ ] ~~MurphPad (Shield: murphpad)~~
- [x] Naked60 (Shield: naked60)
- [ ] Nibble (Shield: nibble) [65%]
- [x] Osprette (Shield: osprette)
- [ ] Pancake (Shield: pancake) [ortho]
- [ ] QAZ (Shield: qaz) [35%]
- [ ] Quefrency Rev. 1 (Shields: quefrency_left, quefrency_right) [split 65%]
- [x] Redox (Shields: redox_left, redox_right)
- [x] REVIUNG41 (Shield: reviung41)
- [ ] ~~Romac+ Macropad (Shield: romac_plus)~~
- [ ] ~~Romac Macropad (Shield: romac)~~
- [ ] SNAP (Shields: snap_left, snap_right) [split 75%]
- [x] Sofle (Shields: sofle_left, sofle_right)
- [x] splitkb.com Aurora Corne (Shields: splitkb_aurora_corne_left, splitkb_aurora_corne_right)
- [x] splitkb.com Aurora Sweep (Shields: splitkb_aurora_sweep_left, splitkb_aurora_sweep_right)
- [x] Splitreus62 (Shields: splitreus62_left, splitreus62_right)
- [ ] TG4x (Shield: tg4x) [40%]
- [ ] ~~Tidbit Numpad (Shield: tidbit)~~
- [ ] ~~2% Milk (Shield: two_percent_milk)~~
- [x] Waterfowl (Shields: waterfowl_left, waterfowl_right) [ergo split]
- [x] Zodiark (Shields: zodiark_left, zodiark_right)

#### MakerDiary nRF52840 M.2 Interconnect

- [x] MakerDiary m60 (Shield: m60) [60%]

#### Seeed XIAO Interconnect

- [x] Hummingbird (Shield: hummingbird) [also for use with rufous]

## Out-of-Tree Keyboards

Keyboards in italics require a fork of ZMK, and the repository is given in parenthesis.

###  Onboard Controller Keyboards

- [x] `ahokore` https://github.com/dezlidezlidezli/ahokore-zmk-config/tree/master/config/boards/arm/ahokore https://github.com/zmkfirmware/zmk/pull/743
- [ ] ~~`adv360` https://github.com/PolarityWorks/zmk-config-adv360/tree/main/config/boards/arm/adv360~~
- [x] `adv360` https://github.com/KinesisCorporation/Adv360-Pro-ZMK/tree/V2.0/config/boards/arm/adv360 (https://github.com/refil/zmk/tree/adv360-z3) [ergo split] [can also use adv360pro]
- [x] `adv360pro` https://github.com/zmkfirmware/zmk/pull/1454 [ergo split] [can also use adv360]
- [x] `ble_chiffre` https://github.com/tominabox1/zmk-config2/tree/master/config/boards/arm/ble_chiffre
- [ ] `ble_chiffre` https://github.com/NCKiser/zmk-config/tree/master/config/boards/arm/ble_chiffre
- [ ] `bt65` https://github.com/PolarityWorks/zmk-config-ckp/tree/bt65-ISO/config/boards/arm/bt65 [65%]
- [ ] `chockea` https://github.com/vattern/zmk-config/tree/master/config/boards/arm/chockea
- [x] _`corne-ish_zen` https://github.com/LOWPROKB/zmk/tree/Board-Corne-ish-Zen-dedicated-work-queue/app/boards/arm/corne-ish_zen (https://github.com/LOWPROKB/zmk/tree/Board-Corne-ish-Zen-dedicated-work-queue)_ [ero split] [can also use corneish_zen_v1]
- [x] `corneish_zen_v1` https://github.com/zmkfirmware/zmk/pull/1593 [ero split] [can also use corne-ish_zen]
- [ ] `cyber60_rev_d` https://github.com/zmkfirmware/zmk/pull/354 [60%]
- [x] `dao` https://github.com/yumagulovrn/dao-zmk-config [ergo split]
- [ ] `draculady` https://github.com/MangoIV/zmk/tree/draculady/app/boards/arm/draculady [ergo split]
- [x] `ffkb-holyiot` https://github.com/sadekbaroudi/zmk-ffkb-holyiot/tree/master/boards/arm/ffkb-holyiot
- [ ] `hakea` https://github.com/vattern/zmk-config/tree/master/config/boards/arm/hakea
- [ ] `hannah60_b` https://github.com/zmkfirmware/zmk/pull/1448 [60%]
- [x] `klacker_bs` https://github.com/weteor/Klacker_BS-Config/tree/main/boards/arm/klacker_bs
- [ ] `le_chiff_ble` https://github.com/MangoIV/le_chiff_ble_config/tree/master/zmk-config/boards/arm/le_chiff_ble [ergo]
- [x] `lpgalaxy_blank_slate` https://github.com/petejohanson/blank-slate-zmk-module/tree/main/boards/arm/lpgalaxy_blank_slate [ortho]
- [x] `one42` https://github.com/cyril279/zmk-config-one42/tree/main/config/boards/arm/one42
- [ ] ~~`paw` https://github.com/zmkfirmware/zmk/pull/774~~
- [ ] `peach` https://github.com/zmkfirmware/zmk/pull/1081 [65%]
- [ ] `polilla_w` https://github.com/zmkfirmware/zmk/pull/612 [ergo]
- [x] `preonic_rev3` https://github.com/zmkfirmware/zmk/pull/228
- [ ] `scylla` https://github.com/vattern/zmk-config/tree/master/config/boards/arm/scylla
- [ ] `slicemk_ergodox` https://github.com/slicemk/zmk/tree/v20210618/app/boards/arm/slicemk_ergodox [ergo split]
- [x] `technikable` https://github.com/petejohanson/technikable-zmk-config/tree/main/boards/arm/technikable
- [x] _`tipper_tf` https://github.com/weteor/Tipper_TF-Config/tree/master/boards/arm/tipper_tf (https://github.com/weteor/zmk/tree/Tipper_TF_rev2)_
- [ ] `tornblue` https://github.com/rtitmuss/zmk/tree/tornblue_r1/app/boards/arm/tornblue [ergo split]
- [x] `zaphod` https://github.com/petejohanson/zaphod-config/tree/main/boards/arm/zaphod

#### Pro Micro Interconnect

- [x] `absolem` https://github.com/mrzealot/zmk-config/tree/master/
- [x] `ahokore` https://github.com/dezlidezlidezli/zmk-config/tree/master/config/boards/shields/ahokore
- [ ] `ajisai` https://github.com/dcored13/zmk-master/tree/main/app/boards/shields/ajisai [ergo]
- [x] `arch36` https://git.sr.ht/~hwrd/zmk-config/tree/main/item/boards/arch36
- [ ] ~~`arch36` https://github.com/okke-formsma/zmk-config/tree/arch36/config/boards/shields/arch36~~
- [ ] `atreus_choc` https://github.com/zmkfirmware/zmk/pull/751 [ergo]
- [ ] `babyvble` https://github.com/zmkfirmware/zmk/pull/967 [ergo]
- [x] `barobord` https://github.com/sadekbaroudi/zmk-barobord/tree/master/config/boards/shields/barobord
- [x] `bastyl` https://github.com/zmkfirmware/zmk/pull/604 [also for use with tbk and scylla]
- [x] `bat43` https://github.com/dnaq/zmk/tree/bat43-keymap/app/boards/shields/bat43
- [ ] `battoota` https://github.com/AlaaSaadAbdo/zmk-config/tree/master/config/boards/shields/battoota [ergo split]
- [x] `berylline` https://github.com/jcmkk3/zmk-config/tree/main/config/boards/shields/berylline
- [ ] ~~`bluehand` https://github.com/zmkfirmware/zmk/pull/970~~
- [ ] `boardsource4x12` https://github.com/alinelena/mlego-zmk/tree/main/config/boards/shields/boardsource4x12 [ortho]
- [ ] ~~`breeze` https://github.com/zmkfirmware/zmk/pull/735~~
- [x] `breeze` https://github.com/zmkfirmware/zmk/pull/1125
- [ ] `breeze` https://github.com/zmkfirmware/zmk/pull/1312
- [ ] `btrfld` https://github.com/SolidHal/zmk/tree/main/app/boards/shields/btrfld [ergo]
- [ ] ~~`cafeBabe` https://github.com/zmkfirmware/zmk/pull/894~~
- [x] `choc-corne` https://github.com/Putitii/zmk-choc_corne/tree/main/config/boards/shields/haploinsufficient [ergo split] [for chocorne36-n!n]
- [ ] `chocofi` https://github.com/Shahabaz-Bagwan/zmk-config/tree/master/config/boards/shields/chocofi [ergo split] [can also use corne]
- [ ] `chocofly` https://github.com/vlkv/chocofly-zmk-config/tree/master/config/boards/shields/chocofly [ergo]
- [ ] ~~`choconum` https://github.com/zmkfirmware/zmk/pull/940~~
- [x] `chocv` https://github.com/brickbots/chocV/tree/main/zmk_config/boards/shields/chocv [also for use with slabv]
- [ ] ~~clueboard_california https://github.com/zmkfirmware/zmk/pull/39~~
- [ ] `dactyl_cc` https://github.com/DarkFox/dactyl-cc-zmk/tree/master/config/boards/shields/dactyl_cc
- [ ] `dactyl_manuform_4x5` https://github.com/zmkfirmware/zmk/pull/760 [ergo split]
- [x] `dilemma` https://github.com/0xcharly/zmk-config/tree/master/config/boards/shields/dilemma [ergo split]
- [x] `draculad` https://github.com/MangoIV/zmk/tree/draculad/app/boards/shields/draculad
- [ ] ~~`duckboard` https://github.com/zmkfirmware/zmk/pull/634~~
- [ ] `duckfoot3` https://github.com/dodubassman/zmk-config-duckfoot-rev3/tree/master/config/boards/shields/duckfoot3 [ergo split]
- [ ] `dumbo` https://github.com/zmkfirmware/zmk/pull/747 [ergo split]
- [ ] ~~`dumbpad` https://github.com/zmkfirmware/zmk/pull/1137~~
- [x] `ergodash` https://github.com/Eyenseo/zmk/tree/main/app/boards/shields/ergodash
- [ ] `ergos1_oe` https://github.com/zmkfirmware/zmk/pull/1333 [ergo split]
- [x] `ergotravel` https://github.com/jchenVT/zmk_ergotravel/tree/main/config/boards/shields/ergotravel
- [ ] `exkeylibur` https://github.com/tlietz/zmk-config/tree/main/config/boards/shields/exkeylibur [ergo split]
- [ ] `faw36` https://github.com/Germain-Gadel/zmk-config-faw36/tree/master/config/boards/shields/faw36 [ergo split]
- [ ] `fifi` https://github.com/zmkfirmware/zmk/pull/1008 [ergo split] [can also use corne]
- [x] `ffkb` https://github.com/sadekbaroudi/zmk-ffkb/tree/master/config/boards/shields/ffkb
- [x] `gatoraid40` https://github.com/freya-irl/zmk/tree/main/app/boards/shields/gatoraid40
- [ ] ~~`gherkin` https://github.com/zmkfirmware/zmk/pull/355~~
- [ ] ~~`gherkin` https://github.com/zmkfirmware/zmk/pull/748~~
- [ ] `helix_4` https://github.com/zmkfirmware/zmk/pull/585 [ortho] [can also use helix]
- [ ] `hhkb` https://github.com/kanru/hhkb-nicenano-zmk/tree/main/config/boards/shields/hhkb [60%]
- [x] `hillside48` https://github.com/mmccoyd/zmk-config/tree/master/config/boards/shields/hillside48 https://github.com/zmkfirmware/zmk/pull/1047
- [ ] `hillside52` https://github.com/mmccoyd/zmk-config/tree/main/config/boards/shields/hillside52 [ergo split]
- [x] `horizon` https://github.com/skarrmann/zmk-config/tree/master/config/boards/shields/horizon https://github.com/zmkfirmware/zmk/pull/1000
- [x] `humla` https://github.com/smores56/zmk-config-humla/tree/main/config/boards/shields/humla [ergo]
- [x] `klor` https://github.com/GEIGEIGEIST/zmk-config-klor/tree/master/config [ergo split]
- [x] `klotz` https://github.com/GEIGEIGEIST/zmk-config-klotz/tree/master/config [ergo split]
- [ ] ~~`launchpad` https://github.com/zmkfirmware/zmk/pull/703~~
- [ ] `lets_split` https://github.com/zmkfirmware/zmk/pull/746 [ortho split]
- [x] `levinson` https://github.com/zmkfirmware/zmk/pull/566
- [ ] `lodash33` https://github.com/zmkfirmware/zmk/pull/283 [ortho 30%]
- [x] `luna` https://github.com/mindhatch/zmk-config-luna/tree/master/config/boards/shields/luna [ergo split]
- [ ] ~~`mach3` https://github.com/zmkfirmware/zmk/pull/1168~~
- [x] `mano42` https://github.com/harshitgoel96/zmk/tree/mano_kb/app/boards/shields/mano42
- [ ] `mercury` https://github.com/jmding8/zmk/tree/mercury_3x5%2B3/app/boards/shields/mercury [split 40%]
- [ ] `micro10` https://github.com/dcpedit/zmk/tree/dcpedit/app/boards/shields/micro10 [30%]
- [x] `microdox_v2` https://github.com/manna-harbour/miryoku/discussions/217 [ergo split]
- [ ] `mokulua` https://github.com/zmkfirmware/zmk/pull/1297 [split 65%]
- [ ] `naked48` https://github.com/kitchen/zmk-config/tree/kitchen/naked48-shield/config/boards/shields/naked48 [ortho]
- [x] `pinkiesout` https://github.com/sadekbaroudi/zmk-pinkiesout/tree/master/config/boards/shields/pinkiesout
- [x] `pockettype` https://github.com/sfgabe/zmk-config-pockettype/tree/master/config/boards/shields/pockettype
- [ ] ~~`pockettype` https://github.com/zmkfirmware/zmk/pull/672~~
- [x] `pteron36` https://github.com/zmkfirmware/zmk/pull/1364
- [x] `rae_dux` https://github.com/andrewjrae/zmk-config/tree/development/config/boards/shields/rae_dux
- [x] `rebound_v4` https://github.com/zmkfirmware/zmk/pull/546
- [ ] `rev1350` https://github.com/cyril279/zmk-config-rev41_1350/tree/main/config/boards/shields/rev1350 [reviung 41 LP] [ergo]
- [ ] `reviung34` https://github.com/zmkfirmware/zmk/pull/1264 [ergo]
- [ ] `reviung34` https://github.com/kitchen/zmk-config/tree/main/config/boards/shields/reviung34 [ergo]
- [x] `reviung39` https://github.com/zmkfirmware/zmk/pull/622
- [ ] ~~`reviung39` https://github.com/zmkfirmware/zmk/pull/695~~
- [ ] ~~`reviung39` https://github.com/zmkfirmware/zmk/pull/795~~
- [ ] `revlp` https://github.com/zmkfirmware/zmk/pull/1482 [ergo]
- [ ] `roadster` https://github.com/zmkfirmware/zmk/pull/768 [40%]
- [x] `rockon` https://github.com/sadekbaroudi/zmk-rockon/tree/master/config/boards/shields/rockon
- [x] `rollow` https://github.com/barbellboards/Rollow-ZMK/tree/barbellboards/rollow/app/boards/shields/rollow [ergo split]
- [x] `scylla_dmmpcb` https://github.com/meln1k/zmk-config/tree/master/config/boards/shields/scylla_dmmpcb
- [ ] `shapeshifter4060` https://github.com/zmkfirmware/zmk/pull/1123 [ergo]
- [ ] `skeletyl_v2_elitec` https://github.com/meln1k/zmk-config/tree/master/config/boards/shields/skeletyl_v2_elitec [ergo split] [can also use tbkmini_adapterv2]
- [x] `sofle_rgb` https://github.com/gregghz/zmk-config/tree/master/config/boards/shields/sofle_rgb
- [ ] `sofle_rgb_keyhive` https://github.com/zmkfirmware/zmk/pull/1233 [ergo split]
- [ ] ~~`soyuz` https://github.com/zmkfirmware/zmk/pull/1073~~
- [ ] ~~`spacepad` https://github.com/zmkfirmware/zmk/pull/785~~
- [ ] ~~`sweet16` https://github.com/zmkfirmware/zmk/pull/1102~~
- [x] `swweeep` https://github.com/sadekbaroudi/zmk-swweeep/tree/master/config/boards/shields/swweeep [ergo split]
- [x] `swoop` https://github.com/adnanaq/swoop-zmk-config [ergo split]
- [ ] `taira` https://github.com/strayer/taira-keyboard-zmk-config/tree/main/config/boards/shields/taira [ergo split]
- [x] `tbkmini` https://github.com/cwebster2/zmk-config/tree/main/config/boards/shields/tbkmini [also for use with skeletyl]
- [x] `tbkmini_adapterv2` https://github.com/karrui/zmk-config-skeletyl-v2/tree/tbkmini-adapterv2/config/boards/shields/tbkmini [also for use with skeletyl]
- [x] `temper` https://github.com/raeedcho/temper-zmk-config/tree/main/config/boards/shields/temper [ergo split] 
- [x] `tightyl` https://github.com/MvEerd/zmk-config/tree/master/config/boards/shields/tightyl
- [ ] ~~`two_percent_milk` https://github.com/zmkfirmware/zmk/pull/684~~
- [ ] `untitled` https://github.com/AnthonyAmanse/zmk-config/tree/main/config/boards/shields/untitled [ergo split]
- [ ] `ut47` https://github.com/zmkfirmware/zmk/pull/641 [40%]
- [x] `viterbi` https://github.com/zmkfirmware/zmk/pull/687
- [ ] `wing36` https://github.com/zmkfirmware/zmk/pull/1388 [ergo split]
- [ ] `yasui` https://github.com/jazemo19/zmk/tree/main/app/boards/shields/yasui [ortho]
- [ ] `yasui` https://github.com/jazemo19/yasui-v2/tree/main/config/boards/shields/yasui [ortho]

#### Seeed XIAO Interconnect

- [x] `chipper` https://github.com/JonMuller/gerbers/tree/main/corne-choc-xiao/zmk_starter/view [ergo split]
- [x] `clog_v2` https://github.com/smores56/clog-v2-zmk-config/tree/main/config/boards/shields/clog_v2 [ergo split, seeeduino_xiao_ble only]
- [x] `klein` https://github.com/snsten/Klein-zmk/tree/main/config/boards/shields/klein [ergo split]
- [x] `kpukboard` https://github.com/manna-harbour/miryoku/discussions/219 [ergo split]
- [x] _`revxlp` https://github.com/petejohanson/revxlp-config/tree/main/config/boards/shields/revxlp (https://github.com/petejohanson/zmk/tree/drivers/gpio-595-shift-register)_ [ergo]
- [x] `rommana` https://github.com/AlaaSaadAbdo/zmk-config/tree/master/config/boards/shields/rommana [ergo split]
- [x] `totem` https://github.com/GEIGEIGEIST/zmk-config-totem/tree/master/config/boards/shields/totem [ergo split]
- [x] `zaphod_lite` https://github.com/petejohanson/zaphod-config/tree/zaphod-lite/boards/shields/zaphod_lite

#### Other

- [ ] `adafruit_feather_nrf52840` https://github.com/zmkfirmware/zmk/pull/1465
- [ ] `bluemicro833` https://github.com/zmkfirmware/zmk/pull/972
- [ ] `bluemacro840_v1` https://github.com/zmkfirmware/zmk/pull/708 
- [x] `microdox_mod` https://github.com/abondis/zmk-config/tree/main/config/boards/shields/microdox_mod
- [ ] `nucleo_wb55cg_dongle`, `zephirum` https://github.com/zmkfirmware/zmk/pull/938
- [x] `sparkfun_nrf52840_mini` https://github.com/abondis/zmk-config/tree/main/config/boards/arm/sparkfun_nrf52840_mini

### ZMK change from Zephyr 3.0 to Zephyr 3.2

Keyboards that worked with ZMK based on Zephyr 3.0 but have not been updated with the [changes required for Zephyr 3.2](https://zmk.dev/blog/2023/04/06/zephyr-3-2#boardshield-changes) can still be built with ZMK based on Zephyr 3.0, but later additions and changes in ZMK will not be available. Use https://github.com/manna-harbour/zmk/tree/zephyr-3.0 as the ZMK fork when building locally, or `manna-harbour/zmk/zephyr-3.0` with the `branches` option when building with the Miryoku ZMK build workflows.

For use with the mouse keys PR, merge https://github.com/manna-harbour/zmk/tree/zephyr-3.0 and https://github.com/urob/zmk/tree/mouse-3.2 as the ZMK fork when building locally, or use `manna-harbour/zmk/zephyr-3.0 urob/zmk/mouse-3.2` with the `branches` option when building with the Miryoku ZMK build workflows.

### ZMK change from Zephyr 2.5 to Zephyr 3.0

Keyboards that worked with ZMK based on Zephyr 2.5 but have not been updated with the [changes required for Zephyr 3.0](https://zmk.dev/blog/2022/04/02/zephyr-3-0#boardshield-changes) can still be built with ZMK based on Zephyr 2.5, but later additions and changes in ZMK will not be available. Use https://github.com/manna-harbour/zmk/tree/zephyr-2.5 as the ZMK fork when building locally, or `manna-harbour/zmk/zephyr-2.5` with the `branches` option when building with the Miryoku ZMK build workflows. For the mousekeys PR, use https://github.com/manna-harbour/zmk/tree/zephyr-2.5+krikun98-mouse-pr / `manna-harbour/zmk/zephyr-2.5+krikun98-mouse-pr`.

### Updating Unmaintained Out-of-Tree Keyboards

To update an out-of-tree keyboard definition:
- If you are able to update the keyboard definition yourself:
  - Fork the original repository
  - Fix the keyboard definition according to the migration guide
  - PR the fixes upstream
  - If there is no response to the PR:
    - [Request](https://github.com/manna-harbour/miryoku/discussions/new?category=ideas) that the repository reference in Miryoku ZMK be updated to your fork
- If you are unable to update the keyboard definition yourself:
  - Create an issue on the original repository
  - If there is no response to the issue:
    - Request help on the [ZMK discord](https://zmk.dev/community/discord/invite)
    - If an updated keyboard definition is made available in another repository:
      - [Request](https://github.com/manna-harbour/miryoku/discussions/new?category=ideas) that the repository reference in Miryoku ZMK be updated

**Comment by bryclee** [comment]
Hi! I'm interested in trying this, but this link https://github.com/manna-harbour/miryoku_zmk#supported-keyboards gives me a 404.

**Reply by manna-harbour** [reply]
Sorry, that link is a little premature. But you're welcome to join the beta now. I just added a link at the top of the document.

**Comment by geeksville** [comment]
I just added an outboard for [this](https://github.com/a741725193/zmk-new_corne) common (and cheap) aliexpress clone "eyelash_corne".  As soon as my keyboard arrives I'll test it and apply any remaining tweaks needed and send in a PR based on [this](https://github.com/manna-harbour/miryoku_zmk/commit/e4fbe12f6294b01143837d61b2594abadbe07f35).
